eos/b-decays/observables.cc

```diff
@@ -66,16 +65,19 @@ namespace eos
             R"(The option "form-factors" selects the form factor parametrization.)",
             {
                 make_observable("B->pilnu::dBR/dq2", R"(d\mathcal{B}(B\to\pi\ell^-\bar\nu)/dq^2)",
-                        &BToPiLeptonNeutrino::differential_branching_ratio,
-                        std::make_tuple("q2")),
+                        &BToPseudoscalarLeptonNeutrino::differential_branching_ratio,
+                        std::make_tuple("q2"),
+                        Options{ { "U", "u" } }),
```

Comment:
Should'nt we fix here also the option "q" = "d" and not rely on the default setting?

Reply:
Nope, we want the user to be able to set it. The options listed here override any user-provided options.